docs/docs/ref/arithx1.md

```diff
@@ -38,116 +38,56 @@ predicates:
-length and are ground at the time evaluation, then VectorResult is the  
-list of numbers of the same length as Left,Right, and such that  
-the nth element of VectorResult is the min of the nth elements of  
-Left and Right, respectively.  
+**`max/3`** If Expr1 and Expr2 are arithmetic expressions which are ground at the
+    time of evaluation, then NumResult is the maximum of the results of
+    evaluation of Expr1 and Expr2 with is/2.
+
+**`min/3`** If Expr1 and Expr2 are arithmetic expressions which are ground at the
+    time of evaluation, then NumResult is the minimum of the results of
+    evaluation of Expr1 and Expr2 with is/2.
+
+**`maximum/2`** If ExprList is a list of arithmetic expressions which are all
+    ground at the time of evaluation, then NumResult is the maximum
+    of the evaluations of all the expressions on ExprList.  If
+    ExprList = [], then NumResult = -0inf.
+
+**`minimum/2`** If ExprList is a list of arithmetic expressions which are all
+    ground at the time of evaluation, then NumResult is the minimum
+    of the evaluations of all the expressions on ExprList.  If
+    ExprList = [], then NumResult = 0inf.
+
+**`sumlist/2`** If ExprList is a list of arithmetic expressions which are all
+    ground at the time of evaluation, then NumResult is the sum
+    of the evaluations of all the numbers on ExprList.
+    If ExprList = [], then NumResult = 0.
+
+**`prodlist/2`** If ExprList is a list of arithmetic expressions which are all
+    ground at the time of evaluation, then NumResult is the product
+    of the evaluations of all the numbers on ExprList.
+    If ExprList = [], then NumResult = 1.
+
+**`sum_squares/2`** If ExprList is a list of arithmetic expressions which are all
+    ground at the time of evaluation, then NumResult is the sum
+    of the squares of the evaluations of all the numbers on ExprList.
+    If ExprList = [], then NumResult = 0.
+
+**`sum_square_diffs/3`** If XList and YList are both lists of numbers and are of the
+    same length, then NumResult is the sum of the squares of the
+    pairwise differences of XList and YList; that is
+    NumResult = (X1-Y1)^2 + (X2-Y2)^2 + ...
+    If XList = YList = [], then NumResult = 0.
+
+**`max_vector/3`** If Left and Right are both lists of numbers which are the same
+    length and are ground at the time evaluation, then VectorResult is the
+    list of numbers of the same length as Left,Right, and such that
+    the nth element of VectorResult is the max of the nth elements of
+    Left and Right, respectively.
+
+**`min_vector/3`** If Left and Right are both lists of numbers which are the same
+    length and are ground at the time evaluation, then VectorResult is the
+    list of numbers of the same length as Left,Right, and such that
+    the nth element of VectorResult is the min of the nth elements of
+    Left and Right, respectively.
 
 ## EXAMPLES
-```
-?- max(3,92,M1), max(3,50-92,M2), max(-6,-7*9,M3), max(23.89, 23.87,M4).
```

Comment:
The examples that you added in 0c5375c32fb2e9b59cb6f90076b5ca8505baf7ae are removed.

Reply:
Fixed by  ba8c12a.